mousestyles/path_diversity/path_features_advanced_proposed.py

```diff
@@ -0,0 +1,71 @@
+from future import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+from mousestyles.path_diversity.path_features import angle_between
```

Comment:
Didn't seem like the angle_between funciton was used in this file. Is it necessary to have it here?